### ISSUE TITLE
Overflow in the body examples

### ISSUE THREAD

**Issue by ibnesayeed** [mulesoft/api-console#287]
Examples block in the body section under each media type does not honor the container width when lines are long. As a result, `Try it` block is pushed out of the container width on the right side. The pre/code block should show horizontal scroll bar when necessary. Here is a code to illustrate it.

``` yml
#%RAML 1.0
title: MemGator Server API
version: 1
baseUri: http://example.com
/hello:
  description: Say Hello
  get:
    responses:
      200:
        body:
          text/plain:
            example: Hello Hello Hello Hello Hello Hello Hello Hello Hello Hello Hello Hello Hello Hello Hello Hello Hello Hello Hello Hello Hello!
```

And here is the effect:

![api-console-overflow](https://cloud.githubusercontent.com/assets/65147/15829986/2a52ab1c-2be4-11e6-9eaa-e2119a4452ee.png)

**Comment by bobmulder** [comment]
Same here, but the developers seems not that active to fix it?

**Comment by sichvoge** [comment]
Hi, we hear you and we will fix this as soon as possible.

**Comment by habitullence** [comment]
Just getting started with RAML and I'm enjoying not writing out all that JSON all the time.

Only this was I noticed this straight away, any ETA?

**Comment by sureshHARDIYA** [comment]
Hi, would be great to have this fixed.

**Comment by carowright** [comment]
Fix available in [v3.0.17](https://github.com/mulesoft/api-console/releases/tag/v3.0.17)
